pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateChannelImpl.java

```diff
@@ -348,7 +413,7 @@ public CompletableFuture<Optional<String>> getOwnerAsync(String serviceUnit) {
         }
 
         ServiceUnitStateData data = tableview.get(serviceUnit);
-        ServiceUnitState state = data == null ? Free : data.state();
+        ServiceUnitState state = data == null ? Init : data.state();
```

Comment:
Maybe we can add a static method to get the service unit state and reuse the method.

Reply:
Updated.